Dockerfile

```diff
@@ -46,7 +46,7 @@ ENV OPENVPN_USERNAME=**None** \
     WEBPROXY_PORT=8888 \
     WEBPROXY_USERNAME= \
     WEBPROXY_PASSWORD= \
-    DOCKER_LOG=false \
+    LOG_TO_STDOUT=false \
```

Comment:
https://github.com/haugene/docker-transmission-openvpn/blob/master/Dockerfile#L140

I think this was removed in merge conflicts